test/TestProjectWithPrePost/TestProjectWithPrePost.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <description>$description$</description>
+    <authors>$authors$</authors>
+    <owners>$authors$</owners>
+    <copyright>$copyright$</copyright>
+    <projectUrl>$projecturl$</projectUrl>
+    <tags>$tags$</tags>
+    <packageTypes>
+      <packageType name="$packagetype$" />
+    </packageTypes>
+    <repository type="git" url="https://github.com/jmezach/MSBuild.Sdk.SqlProj/" />
```

Comment:
Could you replace `jmezach` with `rr-wfm` in this URL?

Reply:
Done